raiden_contracts/deploy/__main__.py

```diff
@@ -109,7 +109,10 @@ def deploy(
             f'Deploying {contract_name} txHash={encode_hex(txhash)}, '
             f'contracts version {self.contract_manager.contracts_version}',
         )
-        receipt = check_succesful_tx(self.web3, txhash, self.wait)
+        (receipt, tx) = check_succesful_tx(self.web3, txhash, self.wait)
+        if not receipt['contractAddress']:  # happens with Parity
+            receipt = dict(receipt)
```

Comment:
`receipt` is sometimes an immutable `AttributeDict`.  That's why I'm making a copy.

Reply:
Ok, then the location is fine.